LiteCore/Query/SQLiteN1QLFunctions.cc

```diff
@@ -1005,6 +1005,40 @@ namespace litecore {
         sqlite3_result_int(ctx, num > 0 ? 1 : (num < 0 ? -1 : 0) );
     }
 
+    static void fl_div(sqlite3_context* ctx, int argc, sqlite3_value **argv) {
+        if (sqlite3_value* mnArg = passMissingOrNull(1, argv); mnArg != nullptr) {
```

Comment:
Follow suit with others and use argc here?  Unlikely to change, but still :)

Reply:
Actually isn't it supposed to be 2, not 1?

Reply:
Copy/paste mistake. Will change it, and another place.